vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -1023,7 +1027,13 @@ unit:AttoA
   qudt:conversionMultiplier 0.000000000000000001 ;
   qudt:conversionMultiplierSN 1.0E-18 ;
   qudt:hasDimensionVector qkdv:A0E1L0I0M0H0T0D0 ;
-  qudt:hasQuantityKind quantitykind:Unknown ;
+  qudt:hasQuantityKind quantitykind:CurrentLinkage ;
+  qudt:hasQuantityKind quantitykind:DisplacementCurrent ;
+  qudt:hasQuantityKind quantitykind:ElectricCurrent ;
+  qudt:hasQuantityKind quantitykind:ElectricCurrentPhasor ;
+  qudt:hasQuantityKind quantitykind:MagneticTension ;
+  qudt:hasQuantityKind quantitykind:MagnetomotiveForce ;
+  qudt:hasQuantityKind quantitykind:TotalCurrent ;
```

Comment:
Question: are there any undesirable entailments following from multiple `hasQuantityKind` properties? 
In particular, is it clear that the values are OR, not AND?

Reply:
I don't see any undesirable implications. The triple is basically saying "This unit is a valid unit when quantifying \<quantitykind\>". It is not an OR.

What our "applicableUnit" algorithm does is use all such triples when computing the applicable units for a given QuantityKind, as described [here](https://github.com/qudt/qudt-public-repo/wiki/Advanced-User-Guide#4-computing-applicable-units-for-a-quantitykind).

Is there something you are worried about?